From e3d18fd1a4534e6639a6c8db17b9f4bc7422d1e7 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Tue, 22 Mar 2022 06:09:22 +0200
Subject: [PATCH] [SW-0] gaudi3: align regs name with specs

MEM_CPY AXUSER for SPI DMA register has been renamed for gaudi3 in
latest global specs update. This patch aligns the regmap with the same.

Change-Id: I8a37b4c3c04ec4ebcf6ca724d8475db0e7d0cf11
---

diff --git a/arch/gaudi3/arc/regmap.h b/arch/gaudi3/arc/regmap.h
index 77387c2..7418e6e 100644
--- a/arch/gaudi3/arc/regmap.h
+++ b/arch/gaudi3/arc/regmap.h
@@ -26,7 +26,7 @@
 #define mmPCIE_PHY_RST_N		mmD0_PSOC_GLOBAL_CONF_PCIE_PHY_RST_N
 #define mmBOOT_STRAP_PINS_L		mmD0_PSOC_BOOT_CONF_BOOT_STRAP_PINS_L
 #define mmPRSTN_MASK			mmD0_PSOC_GLOBAL_CONF_PRSTN_MASK
-#define mmSPI_DMA_AWUSER		mmD0_PARC_GLOBAL_CONF_SPI_DMA_AWUSER
+#define mmSPI_DMA_AWUSER		mmD0_PARC_GLOBAL_CONF_SPY_DMA_HBW_ATTR_CTRL_HB_MMU_BYPASS_OVRD
 #define mmSPI_DMA_CTRL			mmD0_PARC_GLOBAL_CONF_SPI_DMA_CTRL
 #define mmSPI_DMA_STATUS		mmD0_PARC_GLOBAL_CONF_SPI_DMA_STATUS
 #define mmSPI_DMA_DST_ADDR_L		mmD0_PARC_GLOBAL_CONF_SPI_DMA_DST_ADDR_L
